app/models/parliament.rb

```diff
@@ -33,8 +37,14 @@ def last_or_create
     end
   end
 
+  validates_presence_of :dissolution_heading, :dissolution_message, if: :dissolution_at?
+  validates_length_of :dissolution_heading, maximum: 100
+  validates_length_of :dissolution_message, maximum: 600
+
+  after_save { Site.touch }
```

Comment:
Possibly but it also busts the head cache which contains the application.css link